frontend/src/components/OCP/DisplayGrafana.js

```diff
@@ -56,7 +53,7 @@ const getBuildLink = (benchmarkConfigs) => {
 const getGrafanaData = (benchmarkConfigs) => {
     const grafanaURL = "https://grafana.rdu2.scalelab.redhat.com:3000/d/";
     const dashboardKubeBurner = "9qdKt3K4z/kube-burner-report-ocp-wrapper?orgId=1"
-    const dashboardIngress = "nlAhmRyVk/ingress-perf?orgId=1&var-termination=edge&" +
+    const dashboardIngress = "d6105ff8-bc26-4d64-951e-56da771b703d/ingress-perf?orgId=1&var-termination=edge&" +
```

Comment:
Was this const?
If it is a change in the near feature let's pass it by env.

Reply:
It is a constant value, but since we lost our dashboards data recently when recreated the ID changed.

Reply:
ack